package.json

```diff
@@ -13,7 +13,7 @@
   },
   "homepage": "https://github.com/FormidableLabs/builder-react-component",
   "scripts": {
-    "npm:postinstall": "node -e \"require('fs').stat('lib', function(e,s){process.exit(e || !s.isDirectory() ? 1 : 0)})\" || builder run build-lib",
+    "npm:postinstall": "cd lib || builder run build-lib",
```

Comment:
NOTE: Going for @exogen 's shorter, better version of this command.

Let me know if anyone feels we should split out to a separate PR...